MVP/Resources/Translations.resx

```diff
@@ -555,4 +555,7 @@
   <data name="about_credits_title" xml:space="preserve">
     <value>Credits</value>
   </data>
+    <data name="language_spanish" xml:space="preserve">
+      <value>Español</value>

```

Comment:
Do we want to keep the default language values in their own language or default them to their English value @sthewissen? if the case is the latter here we should write "Spanish" instead of "Español".

Reply:
Yeah, in my enthousiasm I decided that it would be cool to show the language name as it is in the current selected language and the name as it is in the language you're picking. Later I realized I can probably extract that info from CultureInfo somehow so those lines will probably be removed in a later PR by me.

Reply:
Alright, sounds good! Everything else seems good so I will proceed to approve these changes. Thanks again for your support @keozx 🥳